src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -900,7 +899,7 @@ void InputProcessor::setObjectItemValue(EnergyPlusData &state,
 
 const json &InputProcessor::getJSONObjectItem(EnergyPlusData &state, std::string_view ObjType, std::string_view ObjName)
 {
-    auto objectInfo = ObjectInfo(std::string{ObjType}, std::string{ObjName});
+    auto objectInfo = ObjectInfo(std::string{ObjType}, std::string{ObjName}); // (THIS_AUTO_OK)
```

Comment:
One thing I didn't catch here, but should have is to change `std::string{ObjType}` to `std::string(ObjType)`. No need to use list constructor notation when simple constructor will do. It's annoying.